configs/sample-input.yaml

```diff
@@ -0,0 +1,29 @@
+horreum_url: https://horreum.foo.bar.com
+horreum_keycloak_url: https://horreum-keycloak.foo.bar.com
```

Comment:
Rather than `bar.com`, I think [accepted practice](https://www.rfc-editor.org/rfc/rfc2606.html#section-3) is to use `example.com`.

Reply:
Done